core/SConscript.unix

```diff
@@ -699,6 +699,8 @@ if FROZEN:
             SOURCE_PY_DIR + 'apps/bitcoin/sign_tx/zcash_v4.py',
         ])
     )
+    if BENCHMARK:
+        SOURCE_PY.extend(Glob(SOURCE_PY_DIR + 'apps/benchmark/*.py'))
```

Comment:
This code was moved only to increase readability, the functionality remains the same.